ui/app/components/auth-jwt.js

```diff
@@ -177,6 +172,14 @@ export default Component.extend({
         e.preventDefault();
       }
       if (!this.isOIDC || !this.role || !this.role.authUrl) {
+        let message = this.errorMessage;
+        if (!this.role) {
+          message = 'Invalid role. Please try again.';
```

Comment:
not sure the use case here, but if there's not role, I would assume the error message would be something like, "Role  required. Please try again." But I could be misunderstanding.

Reply:
Or "Missing role."

Reply:
When you input a role if it is not found for the mount path we need to alert the user. So if for example I input `foo` for role and it's not found then we cannot proceed with requesting the `auth_url` for the oidc authentication.